docs/reference.md

```diff
@@ -447,7 +447,11 @@ The OpenWhisk API supports request-response calls from web clients. OpenWhisk re
 ## System limits
 
 ### Actions
-OpenWhisk has a few system limits, including how much memory an action uses and how many action invocations are allowed per hour. The following table lists the default limits for actions.
+OpenWhisk has a few system limits, including how much memory an action uses and how many action invocations are allowed per minute. 
+
+**Note:** This default limits are for the open source distribution, production deployments like Bluemix likely have higher limits. As an operator or developer you can change some of the limits using ansible inventory variables
```

Comment:
in a second pr you added a blurb on how to change these in ansible - shall we link to it from here?